src/metadata.js

```diff
@@ -80,35 +81,38 @@ const getMetadataFromUri = async (contract, tokenId) => {
     metadataFromUri = await metadataProvider
       .provideMetadata(contract, metadataUri, context)
       .then(({ metadata }) => metadata);
-  } catch { }
+  } catch {}
 
   return metadataFromUri;
 };
 
 const getTokenMetadataFromOffchainView = async (contract, tokenId) => {
   const tzip16Metadata = await getTzip16Metadata(contract);
-  const tokenMetadataView = tzip16Metadata?.views?.find(view => view.name === 'token_metadata');
```

Comment:
👍 Undid the accidental formatting. When you look thoroughly at the codebase, it may look like that no formatting was applied before...

We have to fix this globally in this repo, but let's do it not in terms of a separate task.